src/ai-bundle/src/AiBundle.php

```diff
@@ -611,6 +612,9 @@ private function processAgentConfig(string $name, array $config, ContainerBuilde
                 ->setArguments([
                     $config['prompt']['text'],
                     $includeTools ? new Reference('ai.toolbox.'.$name) : null,
+                    TranslatorInterface::class,
```

Comment:
needs to be a service reference instead:
```suggestion
                    new Reference('translator', ContainerInterface::IGNORE_ON_INVALID_REFERENCE),
```